api.raml

```diff
@@ -122,16 +270,72 @@ schemas:
           }
     responses:
       200:
-        description: |
-          The corresponding user record (matching on `employee_id`) was either
-          created (if it did not yet) or updated to match the given data (if it
-          did exist).
+        description: The new user record was created.
       422:
         description: The given user data does not satisfy some validation rule.
+        body:
+          application/json:
+            schema: Error
+            example: |
+              {
+                "name": "Unprocessable entity",
+                "message": "Employee ID cannot be blank.",
```

Comment:
Since it's just an example, it should be fine.  I didn't consider the fact that it's just one of hundreds of permutations.